docs/concepts/multi-region.md

```diff
@@ -0,0 +1,143 @@
+
+`Peer` objects configures access to other `KubeCarrier` installations for multi region deployments.
+A default `local` Peer will be always present with a `KubeCarrier` installation thats set to `master=true`.
+
+`Peer` objects are Cluster-Scoped.
+
+```yaml
+apiVersion: master.kubecarrier.io/v1alpha1
+kind: Peer
+metadata:
+  name: local
+spec: {}
+status:
+  phase: Ready
+  conditions: []
+  observedGeneration: 0
+  regions: []
+  kubeCarrierVersion:
+    version: v0.2.0
+    #...
+---
+apiVersion: master.kubecarrier.io/v1alpha1
+kind: Peer
+metadata:
+  name: eu-west
+spec:
+  kubeconfigSecret:
+    name: eu-west-kubeconfig
+status:
+  phase: Ready
+  conditions: []
+  observedGeneration: 0
+  regions:
+  - name: eu-west-1
+    provider: loodse
+  - name: eu-west-2
+    provider: loodse
+  - name: eu-west-1
+    provider: not-loodse
+---
+apiVersion: master.kubecarrier.io/v1alpha1
+kind: Peer
+metadata:
+  name: us-east
+spec:
+  kubeconfigSecret:
+    name: us-east-kubeconfig
+status:
+  phase: Ready
+  conditions: []
+  observedGeneration: 0
+  regions:
+  - name: us-east-1
+    provider: loodse
+  - name: us-east-1
+    provider: not-loodse
+```
+
+The `KubeCarrier` object will be extended with a new `master` flag (default False) that will trigger the deployment of the KubeCarrier API server and Multi-Region controllers.
+
+```yaml
+apiVersion: operator.kubecarrier.io/v1alpha1
+kind: KubeCarrier
+metadata:
+  name: kubecarrier
+spec:
+  master: true
+```
+
+KubeCarrier installations with the `master` flag set to true, cannot be registered as a `Peer`.
+
+### Account Management
+
+`Accounts` should me managed in the central Master KubeCarrier installation and are reconciled into all `Peers`. `Accounts` in the master cluster will report additional conditions about the state of this reconciliation into multiple `Peers`. `Accounts` in the master cluster are considered `Unready` when the Account is not present in all `Peers`.
+
+## Components
+
+### Master Controller Manager
+
+The Master Controller Manager checks registered `Peer` objects, coordinates `RegionClaims` and `Accounts` across them.
+
+### API
+
+The KubeCarrier API will be the main integration point for external tooling to interact with KubeCarrier.
+This contains the CLI, Client-Side Operators, Service Catalog Adapters and UIs.
+
+As the central interface to KubeCarrier, it should hide or abstract complexity as much as possible, making it easy to create integrations on top of it.
+
+Features:
+- routes/aggregates requests to/from `Peers`
+- aggregates the same type across multiple `Regions`/`ServiceClusters`
+  - e.g. exposes `CouchDB.eu-west-1.loodse` and `CouchDB.eu-east-1.loodse` as `CouchDB.loodse`
```

Comment:
What about slight version drift issue? That is CRD versions could be the same GVK, but having a slightly different CRD definitions across clusters (e.g. adding single backward compatible field)? IIRC that's the reason we weren't aggregating `ServiceCluster` CRDs into one on single region kubecarrier installation.